ewc-rfc004-individual-wallet-attestation.md

```diff
@@ -22,77 +20,84 @@ Current: OAuth 2.0 Attestation-Based Client Authentication Draft 03 [[1](#60refe
 - [5.0	Implementers](#50implementers)
 - [6.0	References](#60references)
 
 # 1.0	Summary
 
-This specification defines a profile for using OAuth 2.0 Attestation-Based Client Authentication to provide Wallet Unit Attestation as defined within the implementing acts for the context of EWC Pilots. 
+This specification outlines how an individual wallet instance obtains a Wallet Unit Attestation from the Wallet Provider. It employs OAuth 2.0 Attestation-Based Client Authentication to deliver Wallet Unit Attestation to organisational wallets, as defined within the European ARF [1] and the EU Digital Identity Wallet Implementing Acts [2], in the context of EWC Pilots.
 
 # 2.0	Motivation
 
-An EUDI Wallet Instance's life cycle begins when the individual installs an EUDI Wallet app provided by an authorized EUDI Wallet Provider. Once installed, the EUDI Wallet Provider issues a Wallet Unit Attestation (WUA), and the instance is now operational [Ref: ARF 1.4, Chapter 6.5.3]. 
-Using the WUA, an electronice attribute attestation (EAA) provider can authenticate a wallet unit and ensure that it is a valid EUDI wallet that has not been revoked before issuing an EAA.
+An EUDI Wallet Instance lifecycle begins when an individual installs an EUDI Wallet app provided by a certified EUDI Wallet Provider. Upon installation, the EUDI Wallet Provider issues a Wallet Unit Attestation (WUA), enabling the instance to become operational.
 
-An attestation provider must be able to determine if a wallet instance offers the requireed functional capabilities and meets the security requirements for issuance. For this purpose, the attestation provider must receive an attestation of the wallet’s capabilities and proof of non-revocation for the wallet instance.
+Using the WUA, an Electronic Attribute Attestation (EAA) provider can authenticate a wallet unit and verify that it is a valid EUDI wallet that has not been revoked before issuing an EAA.
+
+An attestation provider must be able to determine whether a wallet instance possesses the required functional capabilities and satisfies the security requirements for issuance. To achieve this, the attestation provider must receive proof of the wallet’s capabilities and evidence of non-revocation for the wallet instance.
 
 # 3.0	Wallet Unit Attestation (WUA)
 
-The attestation of a wallet's capabilities through Wallet Unit Attestation is based on the OAuth 2.0 Attestation-Based Client Authentication draft 03[1].
+The attestation of a wallet's capabilities through Wallet Unit Attestation is based on the OAuth 2.0 Attestation-Based Client Authentication draft 03 [3].
 
 ## 3.1 Wallet Unit Attestation JWT
 
-The following Wallet Unit Attestation JWT must be presented when sending the token request as defined in RFC001[7].
+The following Wallet Unit Attestation JWT must be presented when sending the token request defined in RFC001[8] to issuers or when sending the authorisation response as described in RFC002 [9] to relying parties.
```

Comment:
should we include an example of a presentation authorization response? or do you feel these two RFCs combined already have enough info.

Reply:
Good idea. Do you want to suggest an example?

Reply:
I would prefer to add presentation authorization response in a separate PR. Can we keep this one as is just to token request for now? For authorization response may need to consider some extra things.

Reply:
if this addition reference presentation authorisation response  is removed i would approve this PR, and we can deal with presentation authorisation response in a different PR

Reply:
1. So, should we merge this for now and include the example for the authorisation response as a separate addition? 
2. The current approach should suffice for the direct post-response mode in my opinion. Do you agree ?
3. Is your concern about the fragment mode?

Reply:
well i would remove the addition of "or when sending the authorisation response as described in RFC002 [9] to relying parties" and add that together with examples in another PR.

I don't have any specific thoughts about authorization response i just feel like this restructuring PR should just be restructuring and adding  authorization response would be an addition.